store/sqlstore/channel_store.go

```diff
@@ -93,7 +93,7 @@ func (s SqlChannelStore) CreateIndexesIfNotExists() {
 	s.CreateIndexIfNotExists("idx_channelmembers_channel_id", "ChannelMembers", "ChannelId")
 	s.CreateIndexIfNotExists("idx_channelmembers_user_id", "ChannelMembers", "UserId")
 
-	s.CreateFullTextIndexIfNotExists("idx_channels_txt", "Channels", "Name, DisplayName")
+	s.CreateFullTextIndexIfNotExists("idx_channels_txt", "Channels", "Name, DisplayName, Purpose")
```

Comment:
If you change give this index a new name, it will be automatically generated on existing servers.

You'll probably also want to add a `s.RemoveIndexIfExists` to store/sqlstore/upgrade.go so that it gets rid of the old one as well

Reply:
@miguelespinoza did you see the comment from Harrison?

Reply:
@hmhealey thanks for the advice. I'll make those changes today